src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -46,6 +46,179 @@ using namespace ::chip::DeviceLayer::Internal;
 namespace {
 const char kWpaSupplicantServiceName[] = "fi.w1.wpa_supplicant1";
 const char kWpaSupplicantObjectPath[]  = "/fi/w1/wpa_supplicant1";
+
+constexpr uint16_t kWiFi_BAND_2_4_GHZ = 2400;
+constexpr uint16_t kWiFi_BAND_5_0_GHZ = 5000;
+
+static uint16_t Map2400MHz(const uint8_t inChannel)
```

Comment:
static not required: this is already in an anonymous namespace. Here and below.

Reply:
Good catch! acked.